test/scripts/e2e-kind.sh

```diff
@@ -45,7 +48,42 @@ should set default value on new IngressClass
 should prevent Ingress creation if more than 1 IngressClass marked as default
 "
 
-SKIPPED_TESTS=$(echo "${SKIPPED_TESTS}" | sed -e '/^\($\|#\)/d' -e 's/ /\\s/g' | tr '\n' '|' | sed -e 's/|$//')
+IPV4_ONLY_TESTS="
+# Limit the IPv4 related test to IPv4 only deployments
+#  See: https://github.com/leblancd/kube-v6-test
+\[Feature:Networking-IPv4\]
+
+# The following tests currently fail for IPv6 only, but should be passing.
+# They will be removed as they are resolved.
+
+# shard-n Tests
+Network.+should resolve connrection reset issue
+
+# shard-np Tests
+NetworkPolicy.+should allow egress access to server in CIDR block
+
+# shard-s Tests
+Services.+should create endpoints for unready pods
+
+# shard-other Tests
+DNS.+should provide DNS
+DNS.+should resolve DNS
+DNS.+should provide /etc/hosts entries
+"
+IPV6_ONLY_TESTS="
+# Limit the IPv6 related tests to IPv6 only deployments
+#  See: https://github.com/leblancd/kube-v6-test
+\[Feature:Networking-IPv6\]
```

Comment:
I removed this limitation in kubernetes 1.16, since then there is no difference between IPv4 and IPv6 
tests